### ISSUE TITLE
Error opening stream, but sensors and snapshot working

### ISSUE THREAD

**Issue by christiaaneggink** [twrecked/hass-aarlo#394]
Hi @twrecked, 

Big fan of your work :-)

My Arlo Ultra cameras recently stopped streaming in Home Assistant. Strangely enough the sensors and other functionality of the cameras, e.g. snapshots are working as expected. 

I have 2FA working fine using gmail and the Arlo account has Admin access for the shared cameras.

I am running:

```
Home Assistant OS 5.11
Home Assistant supervisor-2021.02.9
Home Assistant core-2021.2.3
HACS 1.11.3
Arlo Camera Support v0.7.1.beta.1
Lovelace Hass Aarlo v0.2-alpha.14
```

I have enabled additional logging in configuration.yaml:

```
logger:
  default: info
  logs:
    custom_components.aarlo: debug
    custom_components.aarlo.camera: debug
    pyaarlo: debug
```

But I only see the followings errors in HA logs:
```
Logger: homeassistant.components.stream.worker
Source: components/stream/worker.py:83
Integration: Stream (documentation, issues)
First occurred: 19:32:47 (4 occurrences)
Last logged: 19:32:56

Error opening stream https://arlostreaming98-z1-prod.wowza.arlo.com:80/stream/A421047WA080C_1613665962456.mpd?egressToken=bcbd5244_376d_406f_8bf8_4cd4e0397ef3&userAgent=web&cameraId=A421047WA080C_1613665962456&txnId=be!3dcd0346-6a3c-4285-83b7-94274e16c353
Error opening stream https://arlostreaming05-z1-prod.wowza.arlo.com:80/stream/A421047AA11FC_1613665967178.mpd?egressToken=b7af1ea4_ec1b_4ff5_94ee_1cf5f5597095&userAgent=web&cameraId=A421047AA11FC_1613665967178&txnId=be!173d181a-df39-4e16-86e0-f2930fc61249
Error opening stream https://arlostreaming08-z1-prod.wowza.arlo.com:80/stream/A421047PA1543_1613665969785.mpd?egressToken=f346c3d5_72d7_4073_b4dd_c08f5e1ab6d3&userAgent=web&cameraId=A421047PA1543_1613665969785&txnId=be!093a6cf7-a89d-4cb0-a0d3-af0d3253ec87
Error opening stream https://arlostreaming1172-z1-prod.wowza.arlo.com:80/stream/A421047LA1346_1613665973184.mpd?egressToken=500dbb3b_7406_4971_982c_a91445341d8f&userAgent=web&cameraId=A421047LA1346_1613665973184&txnId=be!a1b8ff35-1486-4f12-b99e-ed0791a27e7e
```

Any idea what the issue might be?

**Comment by twrecked** [comment]
I recently updated the user-agents and might have broke this. Or it could just be that Arlo changed something.

If you are using the Lovelace card you need to change its config to turn on direct streaming. For the old config:

```
play_direct: true
```

For new config, add `direct` to the `image_view` options.

I'll play around with the user-agents but direct streaming is nicer - it goes straight to Arlo not via Home Assistant and has audio.

**Comment by twrecked** [comment]
It should be fixed with the [hass-aarlo v0.7.1.beta.2](https://github.com/twrecked/hass-aarlo/releases/tag/v0.7.1.beta.2) and [lovelace v0.2-alpha.15](https://github.com/twrecked/lovelace-hass-aarlo/releases/tag/v0.2-alpha.15).

And you shouldn't have to change anything.

**Comment by christiaaneggink** [comment]
First of all, thanks for the extremely speedy response!

Upgrading hass-aarlo integration and lovelace card fixed the problem in Chrome (88.0.4324.150) on my Mac (10.15.7), but the problem seems to persist in Safari on my iPhone 11 Pro (14.3). 

The errors have disappeared from the Home Assistant logs, but I assume that is because it now bypasses the Home Assistant stream component.

**Comment by christiaaneggink** [comment]
Upgrading to lovelace v0.2-alpha.16 has fixed the issue on iPhone and iPad for me. 
Thank you, I will close the issue.
